bot.ml

```diff
@@ -839,7 +839,18 @@ let callback _conn req body =
                         ~number:pr.issue.number
                       >>= function
                       | Ok reviews_info -> (
-                          if not (String.equal reviews_info.baseRef "master")
+                          if
+                            List.exists reviews_info.last_comments ~f:(fun c ->
+                                String.equal comment_info.id c.id
+                                && c.created_by_email)
```

Comment:
I think you should change the logic a bit so that one cannot work around this security check by flooding a PR with comments. The logic should say something like: find the comment with the right ID, if not found post an error message about there being too many comments posted since the merge request (should virtually never happen in practice thankfully). If the comment was found, check it was not created by email.